### PR TITLE
Model database

### PR DESCRIPTION
**Pull request by boo-shehab**
# HI reviewer 👋 

### Project requirements
Be sure to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships.

- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
    - Create and run the necessary migration files.
    - Table and column names should match the ERD diagram.
        - note: photo for users table should be a link to an image
    - Foreign keys should be included.
    - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

**Review by Davidosky007 — COMMENTED** [review]
Hi @boo-shehab 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, with a dummy readme file, kindly customize it to suits your project. Kindly remove microverse logo, fix MIT link, setup instructions, future feature etc

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    t.index ["post_id"], name: "index_likes_on_post_id"
+    t.index ["user_id"], name: "index_likes_on_user_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.string "title"
+    t.string "text"
+    t.integer "comments_counter"
+    t.integer "likes_counter"
+    t.bigint "user_id", null: false
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] The User should be referenced using the `author_id`, not `user_id`. You can refer to [this thread](https://stackoverflow.com/questions/13694654/specifying-column-name-in-a-references-migration) for how to name foreign keys with a different model names

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_03_01_142236) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.string "text"
+    t.bigint "user_id", null: false
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] The user should be referenced with `author_id`, not `user_id`. Kindly fix that.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    t.bigint "post_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id"], name: "index_comments_on_post_id"
+    t.index ["user_id"], name: "index_comments_on_user_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.bigint "user_id", null: false
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] The user should be referenced with `author_id`, not `user_id`. Kindly fix that.

File: README.md [hunk 4]
```diff
@@ -60,57 +60,30 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
```
Comment (julie-ify) [inline README.md:27]: - [ ] Kindly remove the Microverse logo as hinted by the previous reviewer. Or substitute the logo with yours.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @boo-shehab,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @boo-shehab    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**